tests/README.md

```diff
@@ -5,6 +5,7 @@ Be able to test the correctness of the
  - `aim engine`
  - `aim sdk`
  - `aim ql`
+ - extensions
```

Comment:
Wasn't sure what to add here, feel free to suggest a different folder structure.

Reply:
that's completely fine, please just add the marks around as is done with the others